---
layout: photo
title: 测试-python异常
tags:
  - 测试
date: 2022-03-06 18:13:10
categories: 测试
photos:
---
python中异常的处理
<!--more-->
### 在 except 中 return 后执行 finally 中的代码
会继续处理finally中的代码。

### 抛出自定义异常
用raise方法可以抛出自定义异常。

### except 的作用和用法
except: #捕获所有异常 
except: <异常名>: #捕获指定异常
except:<异常名 1, 异常名 2> : 捕获异常 1 或者异常 2 
except:<异常名>,<数据>:捕获指定异常及其附加的数据
except:<异常名 1,异常名 2>:<数据>:捕获异常名 1 或者异常名 2,及附加的数据